CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # OverReact Test Changelog
 
+## 3.0.0
+* Add support for Dart null safety ([#155](https://github.com/Workiva/over_react_test/pull/155))
+  * **BREAKING CHANGES**
+    *  The first argument of `queryByTestId` / `queryAllByTestId` (`root`) is now non-nullable ([#157](https://github.com/Workiva/over_react_test/pull/157)).
```

Comment:
Actually, I don't believe this is a breaking change, since the arg was non-nullable before (and would throw when passed null). From the PR:
> Also, passing null into these methods always threw in v2 and will always throws in v3, so we could also make the args non-nullable.